### DISCUSSION TITLE
zh/posts/acm/bei-bao

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#381] [Announcements]
# zh/posts/acm/bei-bao

1432背包again Gy最近学习了01背包问题，无聊的他又想到了一个新的问题，给定n个物品的价值，和01背包一样，每个物品只能选1次或0次，求最小不能被得到的价值。 输入 第一行一个正整数T(T &lt;= 100)，表示有T组数据。 每组数据输入格式如下： 第一行为一个正整数N(N&lt;=100)，表示物品个数。 第二行N个正整数，表示每个物品...

http://localhost:8080/zh/posts/acm/bei-bao.html

**Comment by Chivas-Regal** [comment]
# 这是男朋友测试评论功能的 markdown 
（www
